src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/LicenseExclusions.txt

```diff
@@ -105,7 +105,7 @@ src/msbuild/src/Build/Instance/ProjectItemInstance.cs|generic-exception
 #
 
 # False positive
-src/nuget-client/build/NOTICES.txt|other-copyleft
+src/nuget-client/build/NOTICES.txt
```

Comment:
What are the other offending licenses here? We should avoid a global exclusion for false positives. That should be reserved for files whose results we don't care about. Multiple licenses can be specified here, separated by commas, as indicated in the comments at the top of the file.

Reply:
There were quite a few:

"apache-2.0 AND (apache-2.0 AND unicode-mappings) AND (unicode-mappings AND python) AND bsd-new AND (mit AND x11) AND x11 AND (lgpl-2.0-plus OR cc-sa-1.0) AND bsd-simplified AND mit AND ms-lpl AND apache-1.1 AND (sax-pd AND mit)"

Reply:
Those are all that are detected by the tool. Some of those are in the list of allowed licenses in the test file. We need to determine which are the new licenses that are not currently allowed.

Reply:
It looks like they're all allowed except for `cc-sa-1.0`. I've made `cc-sa-1.0` an allowed license in https://github.com/dotnet/installer/pull/19185/commits/25e2166d3a8d9dd77b237bdefafcccc930e4d87c